deploy.sh

```
@@ -73,10 +96,102 @@ check_existing_service() {
+    while docker ps --format "{{.Ports}}" | grep -E "(0.0.0.0|\\[::\\]):${port}->" >/dev/null; do
+      port=$((port + 1))
+    done
+    echo "$port"
+  }
+  get_existing_deployment_port() {
+    local ports_line=""
+    ports_line=$(docker ps \
+      --filter "label=com.docker.compose.project=${DEPLOYMENT_NAME}" \
+      --filter "label=com.docker.compose.service=mdv_app" \
+      --format "{{.Ports}}" | head -n 1)
+    if [[ "$ports_line" =~ (0\.0\.0\.0|\[::\]):([0-9]+)->5055 ]]; then
```

Shellcheck (0.11.0)

[info] The mentioned syntax error was in this if expression.

(SC1009)